tripal_feature/theme/templates/tripal_feature_sequence.tpl.php

```diff
@@ -52,7 +52,32 @@
     $sequences_html .= '</div>';
     
   }
-  
+ 
+   // provide spliced mRNA sequences
+   // if this is an mRNA
+  if ($feature->type_id->name == 'mRNA') {
+
+
+      $seqs = tripal_get_feature_sequences(array('feature_id' => $feature->feature_id), array('derive_from_parent' => 1, 'aggregate' => 1,
+       sub_feature_types => array('exon') ,'is_html' => 1, 'width' => $num_bases));
```

Comment:
should this be 'sub_feature_types' ?

Reply:
Fixed in commit 338342a1377463fbfa7d16baf793ed6fd8d0e433